integrations/prodigy_ner_fashion/test_project_prodigy_ner_fashion_brands.py

```diff
@@ -0,0 +1,11 @@
+from spacy.cli.project.run import project_run
```

Comment:
We don't need the test here, since we won't be able to install Prodigy in the CI anyway.